backend/src/interface/image_hoster.rs

```diff
@@ -24,7 +24,10 @@ pub enum ImageHosterError {
     /// Photo not found error
     #[error("the photo id passed was not a valid photo id")]
     PhotoNotFound,
-    /// License invalid
+    /// License invalid. List of valid licenses:
+    /// - `No known copyright restrictions`
+    /// - `Public Domain Dedication (CC0)`
+    /// - `Public Domain Mark`
```

Comment:
a thought of putting these *inside* the error message as this would be helpful for api users and be not redundant when we just use the variable.